specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -12806,6 +15566,90 @@
         }
       }
     },
+    "ContinuousWebJob": {
+      "description": "Continuous Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ContinuousWebJob resource specific properties",
+          "properties": {
+            "status": {
+              "description": "Job status.",
+              "type": "string"
+            },
+            "detailedStatus": {
+              "description": "Detailed status.",
+              "type": "string"
+            },
+            "logUrl": {
+              "description": "Log URL.",
+              "type": "string"
+            },
+            "name": {
+              "description": "Job name. Used as job identifier in ARM resource URI.",
+              "type": "string"
+            },
+            "runCommand": {
+              "description": "Run command.",
+              "type": "string"
+            },
+            "url": {
+              "description": "Job URL.",
+              "type": "string"
+            },
+            "extraInfoUrl": {
+              "description": "Extra Info URL.",
+              "type": "string"
+            },
+            "jobType": {
```

Comment:
@ Eric, Is there a finite set of known valid values for this type? If so, please decorate with something like. 
[PropertySettings(AllowedStringValues = new[] { "Value1", "Value2", "Value3" }, ModelValuesAsString = true, AllowedStringValueCollectionName = "WebJobType")]


---
In reply to: [134406455](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r134406455) [](ancestors = 134406455)